From: Steve Dickson <SteveD@redhat.com>
Date: Fri, 12 Feb 2010 15:41:44 -0500
Subject: [nfs] Remove a redundant check for PageFsCache in nfs_migrate_page()
Message-id: <4B7576B8.3000006@RedHat.com>
Patchwork-id: 23250
O-Subject: [RHEL6] [PATCH 2/2]  NFS: Remove a redundant check for PageFsCache
	in nfs_migrate_page()
Bugzilla: 563938

commit 7549ad5f9b6eda49bbac4b14c5b8f37bf464f922
Author: Trond Myklebust <Trond.Myklebust@netapp.com>
Date:   Mon Feb 8 09:32:34 2010 -0500

    NFS: Remove a redundant check for PageFsCache in nfs_migrate_page()

    Signed-off-by: Trond Myklebust <Trond.Myklebust@netapp.com>
    Acked-by: David Howells <dhowells@redhat.com>

Signed-off-by: Aristeu Rozanski <arozansk@redhat.com>

diff --git a/fs/nfs/write.c b/fs/nfs/write.c
index f8aea47..977a1a4 100644
--- a/fs/nfs/write.c
+++ b/fs/nfs/write.c
@@ -1598,8 +1598,7 @@ int nfs_migrate_page(struct address_space *mapping, struct page *newpage,
 	struct nfs_page *req;
 	int ret;
 
-	if (PageFsCache(page))
-		nfs_fscache_release_page(page, GFP_KERNEL);
+	nfs_fscache_release_page(page, GFP_KERNEL);
 
 	req = nfs_find_and_lock_request(page);
 	ret = PTR_ERR(req);
